.github/workflows/main-pr.yaml

```diff
@@ -51,19 +51,17 @@ jobs:
         run: ct install --all
 
   integration-test:
-    # Not a real requirement, but due to worker CPU contention, this can fail if it runs concurrent with lint-test
-    needs: lint-test
```

Comment:
Since jobs are run an undetermined set of workers (if we're not using self hosted workers, which is the case for now) then we shouldn't worry about this right?

Reply:
Unsure. IIRC GitHub Actions workers can get ornery when running multiple KIND instances simultaneously, which is what I think this was for. On the other hand, GitHub presumably upgrades their own infrastructure, maybe this runs fine now without the block. We'll see if tests start failing in weird ways, but probably fine to try without it.